spec/models/call_session_spec.rb

```diff
@@ -1,15 +1,10 @@
 require "rails_helper"
-require "mock_redis"
 
 describe CallSession, type: :model do
   let!(:user) { create(:user, :with_sanitized_phone_number) }
   let!(:patient) { create(:patient, :with_sanitized_phone_number) }
   let!(:call_id) { SecureRandom.uuid }
 
-  before do
-    allow(CallSessionStore).to receive(:create_redis).with(anything).and_return(MockRedis.new)
-  end
```

Comment:
This mocking wasn't necessary, we can use the real Redis for these specs (which revealed the underlying deprecation before I fixed the pipelined call).

Reply:
Oops nevermind, we need mock_redis on ci.